src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/createwelshnotice/CreateWelshNoticeSubmittedHandlerTest.java

```diff
@@ -74,14 +88,17 @@ public void shouldCallUpdateCaseWithCorrectEvent() {
         SscsCaseData caseData = callback.getCaseDetails().getCaseData();
         IdamTokens idamTokens = IdamTokens.builder().build();
         when(idamService.getIdamTokens()).thenReturn(idamTokens);
-        when(ccdService.updateCase(caseData, callback.getCaseDetails().getId(), EventType.DIRECTION_ISSUED_WELSH.getCcdType(),
-            "Create Welsh notice",
-            "Create Welsh notice", idamTokens))
-            .thenReturn(SscsCaseDetails.builder().data(SscsCaseData.builder().build()).build());
+
+        given(updateCcdCaseService.triggerCaseEventV2(anyLong(), eq(DIRECTION_ISSUED_WELSH.getCcdType()), anyString(),
+            anyString(), eq(idamTokens)))
+            .willReturn(SscsCaseDetails.builder().data(caseData).build());
 
         handler.handle(SUBMITTED, callback, USER_AUTHORISATION);
-        verify(ccdService).updateCase(caseData, callback.getCaseDetails().getId(), EventType.DIRECTION_ISSUED_WELSH.getCcdType(),
-            "Create Welsh notice", "Create Welsh notice", idamTokens);
+
+        then(updateCcdCaseService).should(times(1))
+            .triggerCaseEventV2(anyLong(), eq(DIRECTION_ISSUED_WELSH.getCcdType()), anyString(),
+                anyString(), eq(idamTokens));
+
```

Comment:
Is it worth making this a tighter verification using eq(callback.getCaseDetails().getId()) and similar for event summary and description.